pylint.rc

```diff
@@ -0,0 +1,91 @@
+# Version of this pylint.rc file
```

Comment:
This file **does not** belong here! It's in the Organization repo. I can see how the README for Organization repo is confusing, it will be corrected.

Reply:
I updated the documentation and made a PR